src/main/java/com/uid2/optout/Main.java

```diff
@@ -116,8 +118,12 @@ public Main(Vertx vertx, JsonObject config) throws Exception {
         final DownloadCloudStorage contentStorage;
         if (coreAttestUrl != null) {
             String coreApiToken = this.config.getString(Const.Config.CoreApiTokenProp);
+            if (coreApiToken == null || coreApiToken.isBlank()) {
+                LOGGER.warn("Core Attest URL set, but the API Token is not set");
```

Comment:
What does this mean? Is this an error case?

Reply:
This is to try and help figure out why attestation is failing. I want to see if the key is set